cmd/minishift/cmd/addon/addon_apply.go

```diff
@@ -40,8 +40,8 @@ const (
 
 var addonsApplyCmd = &cobra.Command{
 	Use:   "apply ADDON_NAME ...",
-	Short: "Executes the specified add-ons.",
-	Long:  "Executes the specified add-ons. The command works with both enabled and disabled add-ons.",
+	Short: "Applies the specified add-ons.",
+	Long:  "Applies the specified add-ons. You can run this command on enabled or disabled add-ons.",
```

Comment:
@LalatenduMohanty none of what you're saying is clear from the original text. This information wasn't available to me when I edited this string, and most of the time the word "and" is incorrectly used when "or" should be used (meaning, you can run it on "any").

Now that you added this new information, I suggest the following: "You can specify more than one add-on to apply. The add-ons can be enabled or disabled."

This way it's more clear that you can apply multiple add-ons and they can have different states. WDYT?